src/@ndlib/gatsby-theme-marble/components/Layout/PageWrapper/index.js

```diff
@@ -4,18 +4,26 @@ import SkipToMain from '@ndlib/gatsby-theme-marble/src/components/Layout/PageWra
 import NavigationHeader from '@ndlib/gatsby-theme-marble/src/components/Layout/PageWrapper/NavigationHeader'
 import Footer from './Footer'
 import FeedbackModal from '../../Internal/FeedbackModal'
+import BetaModal from '../../Internal/BetaModal'
 
 const PageWrapper = ({ children, location }) => {
   const noFeedbackModal = ['/help/contact-us']
   const feedback = noFeedbackModal.includes(location.pathname) ? (
     null) : <FeedbackModal />
+  let seenWarning
+  if (typeof window !== 'undefined' && location.pathname === '/') {
+    seenWarning = localStorage.getItem('seenWarning')
+  }
+  const betaPopup = seenWarning ? (
+    null) : <BetaModal />
```

Comment:
This is going to work in development, but not in production. `seenWarning` defaults to null which will include the BetaPopup in the pre-rendered code.

Reply:
I added a fix for the breaking error: https://github.com/ndlib/marble-web/pull/34/commits/f2ef25da362f984d925c88efdddf08042699f004

**Explanation:**
So by defaulting to `true` on `seenWarning` BetaPopup won’t render during the build process, but when the page is loaded into the browser it will still do the check for lines 13-15 and show the popup if the localStorage setting does not exist.

**Additional Change**
Can we get a little more space between the line with the checkbox and the button? I'm okay with adding a line of `<div>&nbsp;</div>` between the two.

![Screen Shot 2021-02-22 at 11 51 58 AM](https://user-images.githubusercontent.com/416559/108741120-60d7b380-7504-11eb-8059-d96aaf24b4c7.png)

Reply:
Great! Thanks dan.